brabbuss/overlook
# Dashboard/manager

### What does this PR do?
#####UI
###### complete toolbar and reveal in appropriate contexts:
- 522e118
- 4148962
- bb6dab6
- d50dd98
- b7bdcd0

#####UX 
###### add empty states and styling (hover, copy, etc.) to help user flow
- 4148962
- 1b408f4
- 2e59164
- f2e48f2
- 8c06d57

###### add sidebar with user/manager stats
- 6a54d53

##### Logic
###### add dynamically updating room list and user/manager stats
- 3ae869c
- 017663e
- e46a171

##### Meet spec
###### other changes include:
- c8da8cc  add book/delete on behalf of customer for manager dashboard
- numerous bug fixes: 760b378 83cc468 017663e e2cb554 db474dd 7e35084

### What are the relevant tickets (please link)?
#27, #26, #23, #22, #21, #20, #16, #15, #13, #12

### How should this be tested?
1. At this point, user can try logging in and going throughtout the entire website. Should see updates as the user takes actions. Should be relatively self-explainatory. Only one dead link on the home page (rooms)
